testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -10,6 +10,14 @@ use stacks::burnchains::bitcoin::{BitcoinNetworkType,
 use super::RunLoopCallbacks;
 
 /// Coordinating a node running in neon mode.
+#[cfg(test)]
+pub struct RunLoop {
+    config: Config,
+    pub callbacks: RunLoopCallbacks,
```

Comment:
Not part of this PR, but what’s our take on reintroducing these callbacks on Neon? They were pretty handy for orchestrating tests.

Reply:
I'd be supportive of it, though I think the callbacks may end up having slightly different timing in Neon